CHANGELOG.md

```diff
@@ -10,27 +26,25 @@
 - configurable REMOTE\_USER\_ENVIRON\_NAME [\#58](https://github.com/theforeman/puppet-pulpcore/issues/58)
 - Pulp 2 and 3 API calls through Apache don't play nice [\#49](https://github.com/theforeman/puppet-pulpcore/issues/49)
 - Add smoke tests to nightly installer pipelines [\#42](https://github.com/theforeman/puppet-pulpcore/issues/42)
-- Update to use django-admin solution for pulpcore [\#39](https://github.com/theforeman/puppet-pulpcore/issues/39)
```

Comment:
why did the old changelog re-sort?

Reply:
Not sure, but it was consistent when I reran the command a few times.

Reply:
kk